tools/cldr-apps/.settings/org.eclipse.wst.common.component

```diff
@@ -1,35 +1,48 @@
 <?xml version="1.0" encoding="UTF-8"?><project-modules id="moduleCoreId" project-version="1.5.0">
-            
+                
```

Comment:
Eclipse insert extra blank lines when a configuration file is updated. This is probably a bug in Eclipse. You may get rid of these unnecessary blank lines inserted automatically.

Reply:
yeah. I was going to ignore these files originally, but I think we should just let them get updated…